spanner/google/cloud/spanner/__init__.py

```diff
@@ -15,6 +15,10 @@
 """Cloud Spanner API package."""
 
 
+import pkg_resources
+__version__ = pkg_resources.get_distribution('google-cloud-spanner').version
```

Comment:
Yeah, that was my plan.

`__version__ =` has to come where it does or the other modules are unable to `from google.cloud.spanner import __version__` without a circular import.